add October 2019 newsletter

cosden opened the pull request:
This adds the October 2019 newsletter to archive on the website.
It's a copy/paste conversion to markdown. 

## Checklist:
- [x] I have posted the link for the PR in the usrse slack (#website) to ask for reviewers
- [x] I have previewed changes locally
- [  ] I have updated the CHANGELOG and (if necessary) the README.md
(I didn't think this qualified as changelog worthy)

cc @usrse-maintainers

vsoch commented on _posts/newsletters/2019-10-24-newsletter.md:
```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: October 2019 US-RSE Newsletter
+subtitle: US-RSE Newsletter
```
Maybe for title we should do “RSE Newsletter” and then for the subtitle have the date?

cosden replied:
Good suggestion.  It will look better on the archive page too.

vsoch commented on _posts/newsletters/2019-10-24-newsletter.md:
```diff
@@ -0,0 +1,155 @@
+layout: post
+title: October 2019 US-RSE Newsletter
+subtitle: US-RSE Newsletter
+category: newsletter
```
We should stay away from tags with spaces until we can be sure to url_encode all the places they might be linked. All lowercase with hyphens would work for now.

cosden replied:
Easy fix.

vsoch commented on _posts/newsletters/2019-10-24-newsletter.md:
```diff
@@ -0,0 +1,155 @@
+
+[Gateways 2019](https://sciencegateways.org/web/gateways2019) took
+place in September in San Diego and the conference offered Learning
+Labs - conceptual a mixture of BoFs and break-out sessions - to spawn
```
Conceptually?

cosden replied:
Oops.

vsoch commented on _posts/newsletters/2019-10-24-newsletter.md:
```diff
@@ -0,0 +1,155 @@
+mainly around how US-RSE can support career paths in the US and help
+with creating positions at institutions. One take-away is that
+templates for job descriptions would be helpful for managers offering
+RSE-like positions.
```
I have the one hat Ruth and I wrote in 2016, I could see if I could share it. (Where?)

cosden replied:
I have a bunch too.  We'll need to figure it out (and any ramifications to sharing)

vsoch commented on _posts/newsletters/2019-10-24-newsletter.md:
```diff
@@ -0,0 +1,155 @@
+[Research Software Engineer/Senior Research Software Engineer](https://brown.wd5.myworkdayjobs.com/en-US/staff-careers-brown/job/180-George-Street/Research-Software-Engineer-Senior-Research-Software-Engineer_REQ162388): Brown University
+
+## RSE Stories
+
```
😀❤️🦖🥑🤓

vsoch commented on _posts/newsletters/2019-10-24-newsletter.md:
```diff
@@ -0,0 +1,155 @@
+
+The [Research Software Engineer
+Stories](https://us-rse.org/rse-stories/) is a new interview-style
+podcast designed to spotlight the diverse and interesting people
```
Episode 4 is mostly done and 5 is underway! For 5 I’m testing an actual storytelling format, the goal to also be able to tel the stories of those that might be too introverted to talk on a podcast. Stay tuned!

vsoch commented:
okay taking a look! At a real computer this time, and not half asleep :)

vsoch commented:
Looks great! Even better than the email, actually:

https://272-120906408-gh.circle-artifacts.com/0/usrse.github.io/2019-10-24-newsletter/index.html

Question - the community call links to a [calendar](https://calendar.google.com/calendar/embed?src=kgh79lg13k1d8a2o1s6megfuhc%40group.calendar.google.com&ctz=America%2FNew_York), but it doesn't provide any link or information about how to even join that. I didn't even know there was such a thing! Is there a way to share that, so that the community knows how to participate? That url also doesn't work to import into Google calendar, so it would be good to share a link that is import-able. Actually, you can click the bottom right to add it - that should be better advertised because it's not super obvious.

For the job postings, do you want to have a list, or as is?

![image](https://user-images.githubusercontent.com/814322/67573889-5dad9700-f707-11e9-863a-32a651f455d3.png)

I'm not sure if you want the slack link to be a markdown link or just in parens?

> US-RSE Slack Workspace(https://usrse.slack.com).

hahahhaa okay I'm okay with leaving this :)

> Vanessa Saurus created a nice tool to scoring and displaying how well a repository meets some reproducibility goals and best practices called The Software Checklist.

cosden commented:
We'll need to do a better job with the calendar moving forward!  I'll tag Chris over there. 

I just fixed the job posting to be a list.  And fixed the link to the slack workspace.  Both were goofs on my end when moving to markdown.  Thanks for catching @vsoch!
